docs/business-finder-link-based-facets.md

```diff
@@ -0,0 +1,51 @@
+# EU Exit Business Readiness finder link-based facets
+
+This document explains why the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) ("business finder") uses [content-tagger](https://github.com/alphagov/content-tagger) 'links' to filter the finder.
```

Comment:
I think you can drop this paragraphs. Content-tagger is just the publisher.

I think an example of each would be good. e.g. a screenshot of a content item showing "normal" facets in the `details` and another showing "link-based-facets".

Reply:
I've reworded it, let me know what you think. I think it's helpful to frontload the document with a summary of how the business finder is different to other finders.

Reply:
The content tagger part can be dropped though, as they're not content-tagger links, they're just links. It's important to know that finder-frontend reads the facets from links for the business finder, but now how those facets got there.

e.g.

This document explains why the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) ("business finder") 'links' to filter the finder.

Also, if this section really is the "why" and not the "what"/"how" it should be in an ADR. Happy to talk about this in person.